parlai/crowdsourcing/tasks/model_chat/analysis/compile_results.py

```diff
@@ -15,6 +15,7 @@
 
 from parlai.crowdsourcing.utils.acceptability import AcceptabilityChecker
 from parlai.crowdsourcing.utils.analysis import AbstractTurnAnnotationResultsCompiler
+from parlai.crowdsourcing.tasks.model_chat.model_chat_blueprint import BLUEPRINT_TYPE
```

Comment:
To get rid of this lint error, you can add the `_ = BLUEPRINT_TYPE` line that exists in the per-turn eval analysis script

Reply:
why do we need `--task-name`? Where is this used?

Reply:
Hmm - you need to specify it for use with https://github.com/facebookresearch/ParlAI/blob/7660197631c06ee41c0ce25c5de37a74b11ecb77/parlai/crowdsourcing/utils/analysis.py#L107 , which your code now calls via `self.get_task_data()`